gen-ai-rag-vertex-ai-vector-search/README.md

```diff
@@ -9,9 +9,21 @@ Center](https://cloud.google.com/architecture/).
 To try this sample:
 
   1. [Enable the required APIs](https://console.cloud.google.com/flows/enableapi?apiid=run.googleapis.com,pubsub.googleapis.com,aiplatform.googleapis.com,iam.googleapis.com,storage.googleapis.com,cloudfunctions.googleapis.com,eventarc.googleapis.com,cloudbuild.googleapis.com).
-  1. Follow the [setup instructions](/README.md#setup).
-  1. Make changes to the sample, as indicated by `Note: ` comments.
+  1. Follow the [setup instructions](/README.md#setup) to deploy the sample.
 
+This will deploy the supporting infrastructure for the pattern.
+
+If you want to install a functional example application, follow the
+instructions in the [RAG Architectures - App Source
+Code](https://github.com/GoogleCloudPlatform/devrel-demos/tree/main/ai-ml/rag-architectures).
+
+> [!NOTE]
+> The App Source Code was produced for a Google Cloud Next demo and is unmaintained.
+
+We suggest manually changing the deployed infrastructure to try this example
```

Comment:
Maybe update to say to try this linked example application (because THIS example could also mean the one linked here which is what I got confused about)

Reply:
Updated words to reflect which "this"